app/views/users/_web_monetization.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if policy(Article).create? %>
+<div class="crayons-card crayons-card--content-rows">
```

Comment:
(non-blocking): Shall we indent this line?

Reply:
Yeah, I was wondering how the diff would render from a partial file move.  But since there's no diff, I'll amend.